userinfo-api-faq.md

```diff
@@ -0,0 +1,143 @@
+<!-- START_METADATA
+---
+title: Userinfo API frequently asked questions
+sidebar_label: FAQ
+sidebar_position: 50
+description: Userinfo API frequently asked questions
+pagination_next: null
+pagination_prev: null
+---
+END_METADATA -->
+
+# Frequently asked questions
+
+Here are the Userinfo API Frequently Asked Questions (FAQ).
+See the [Vipps Userinfo API](README.md) for all the technical details.
+
+For more common Vipps questions, see:
+
+* [Vipps API General FAQ](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/faqs)
+
+
+<!-- START_COMMENT -->
+
+ℹ️ Please use the website:
+[Vipps MobilePay Technical Documentation](https://vippsas.github.io/vipps-developer-docs/docs/APIs/userinfo-api).
+
+<!-- END_COMMENT -->
+
+## Which scopes can I use? Why do I get `Invalid_scope`?
+
+If you get “Invalid_scope” this means that you have included one or more scopes
+that you do not have access to or that is not supported.
+See:
+[Scopes](api-guide/core-concepts.md#scopes).
```

Comment:
invalid_scope is a very login specific error code

Reply:
I will move it back to login.